notebooks/official/pipelines/custom_tabular_classification_bq_io_pipeline.ipynb

```diff
@@ -0,0 +1,1461 @@
+{
```

Comment:
<p>remove the 'return'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/992/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>

Reply:
done